clean.go

```diff
@@ -62,7 +62,12 @@ func isLicenseFile(path string) bool {
 }
 
 func isVendorConfFile(path string) bool {
```

Comment:
thought about changing the name, but `go.mod` is also a "vendor configuration file", so 🤷‍♂